File: go/extra_recurring_plan_fee.go
// this file is @generated
package meteroid

// Extra recurring fee
type ExtraRecurringPlanFee struct {
	BillingType BillingType `json:"billing_type"`

	Cadence BillingPeriodEnum `json:"cadence"`

	Quantity int32 `json:"quantity"`

	UnitPrice string `json:"unit_price"`
}

File: python/meteroid/models/extra_recurring_plan_fee.py
# this file is @generated
import dataclasses
from decimal import Decimal

from ..serialization import BaseModel
from .billing_period_enum import BillingPeriodEnum
from .billing_type import BillingType


@dataclasses.dataclass
class ExtraRecurringPlanFee(BaseModel):
    """Extra recurring fee"""

    billing_type: BillingType

    cadence: BillingPeriodEnum

    quantity: int

    unit_price: Decimal
